modules/ROOT/pages/http/http-post-form-task.adoc

```diff
@@ -106,7 +98,10 @@ Note that `yearborn` is one of the form parameters that is expected from incomin
 ----
 
 
-== Requests with a x-www-form-urlencoded Type Body - Full XML Code
+== Post Form by Sending HTTP Requests
+To send an HTTP request that contains form parameters, you can use an API client tool such as https://www.postman.com/product/api-client[Postman], or the http://curl.haxx.se/[curl] command line utility.  If you use Postman, make sure you send the form parameters with content-type `x-www-form-urlencoded`.
```

Comment:
So...are we sending a request that contains form parameters OR posting a form by sending a request?
Clarify; use the same words and word order throughout to avoid confusion.